pkg/move/technologies_test.go

```diff
@@ -219,29 +236,38 @@ func TestFilterFilesByTechnology(t *testing.T) {
 		}, paths)
 	})
 	t.Run("not filter non-existing technology", func(t *testing.T) {
-		paths, err := filterFilesByTechnology(testLog, fs, sourceDir, []string{"php"})
+		tmpDir := t.TempDir()
+
+		sourceDir := filepath.Join(tmpDir, testSourceDir)
+		_ = os.MkdirAll(sourceDir, 0755)
+		_ = os.WriteFile(filepath.Join(sourceDir, "manifest.json"), []byte(manifestContent), 0600)
+
+		paths, err := filterFilesByTechnology(testLog, sourceDir, []string{"php"})
 		require.NoError(t, err)
 		assert.Empty(t, paths)
 	})
 	t.Run("filter with missing manifest", func(t *testing.T) {
-		fs := afero.Afero{Fs: afero.NewMemMapFs()}
-		paths, err := filterFilesByTechnology(testLog, fs, sourceDir, []string{"java"})
+		tmpDir := t.TempDir()
+
+		sourceDir := filepath.Join(tmpDir, testSourceDir)
+		_ = os.MkdirAll(sourceDir, 0755)
+
+		paths, err := filterFilesByTechnology(testLog, sourceDir, []string{"java"})
 		require.Error(t, err)
 		assert.Nil(t, paths)
 	})
 }
 
-func assertFileExists(t *testing.T, fs afero.Fs, path string) {
+func assertFileExists(t *testing.T, path string) {
```

Comment:
these helpers can be replaced with testify

Reply:
https://pkg.go.dev/github.com/stretchr/testify/assert#FileExists